usr/initiator.c

```diff
@@ -1150,8 +1150,11 @@ static void iscsi_stop(void *data)
 	iscsi_ev_context_put(ev_context);
 
 	if (!(conn->session->t->caps & CAP_LOGIN_OFFLOAD)) {
-		if (!iscsi_send_logout(conn))
+		rc = iscsi_send_logout(conn);
+		if (!rc)
 			return;
+		conn_error(conn, "Could not send logout pdu(%s) from iscsi_stop."
+			"Dropping session", strerror(rc));
```

Comment:
Nit: see below: no need to break this into two lines, and you missed a space between sentences because of it.